RELEASE_NOTES.md

```diff
@@ -41,6 +43,7 @@ For a list of all features with an announced removal target, see [Removals](intr
 
 ## Known issues
 
+* CSM 1.6.0 does not support servers with NVIDIA CPUs and GPUs. Systems with these servers should not be upgraded to CSM 1.6.0.
```

Comment:
```suggestion
* CSM 1.6.0 **does not support servers with NVIDIA CPUs and GPUs**. Systems with these servers should not be upgraded to CSM 1.6.0.
```

Reply:
Thanks, I made this change, on a new commit.